tests/integration/ambient/main_test.go

```diff
@@ -55,17 +55,23 @@ var (
 
 type EchoDeployments struct {
 	// Namespace echo apps will be deployed
-	Namespace         namespace.Instance
-	Waypoint          echo.Instances
-	Captured          echo.Instances
-	Uncaptured        echo.Instances
-	SidecarWaypoint   echo.Instances
-	SidecarCaptured   echo.Instances
+	Namespace namespace.Instance
+	// Waypoint echo service
```

Comment:
If we are going to annotate these with comments, I would prefer if they are more explicit about what "kind" of echo service this is - e.g. 

- `// Traffic-captured echo services configured with L7 waypoint proxies`
- `// Traffic-captured echo services`
- `// Traffic-captured echo services with an injected sidecar`

etc etc

Reply:
https://github.com/istio/istio/pull/49353/commits/1b86c5bbc2595e6c20907597592419e51f2176e6